Shorten the bug report template in order to maximize attention (#88158)
godotengine/godot: CONTRIBUTING.md

call a _regression_), please mention it and clarify which versions you tested
(both the one(s) working and the one(s) exhibiting the bug).

### Sharing system information

In the bug report, it's important to share your system information, specifically
the OS version, and when relevant hardware information.

**Bug reports not including the required information may be closed at the 
maintainers' discretion.** If in doubt, always include all the requested 
information; it's better to include too much information than not enough 
information.

Since Godot 4.1, you just have to copy and paste system information by clicking 
on *Help > Copy System Info* at the top of the editor window.

<details>
  <summary>For Godot 4.0 and prior</summary>

  Prior to Godot 4.1, the *Help > Copy System Info* button doesn't exist. So you

>   must find and share in the bug report these elements:

Reviewer: ```suggestion
  must manually collect and share the following with us:
```
